storybook/src/example-pages/toepassingen/regels-op-de-kaart/documenten-VRK.content.ts

```diff
@@ -74,6 +74,7 @@ export const cardList: CardContainer<TemplateResult> = {
           children: html`Test informatie`,
         },
       ],
+      dsoCardClick() {},
```

Comment:
Nou, ik had jou helemaal niet ingeschat als een ouder die zo goed naar z'n kind luistert 🤷

Reply:
Is weg, want optioneel in model

Reply:
🤪